src/components/dashboard/orders/OrderEdit.js

```diff
@@ -0,0 +1,84 @@
+import React, {Component, PropTypes} from 'react';
+import Select from 'react-select';
+import FileSelector from '../../FileSelector';
+import SuccessMessage from '../../SuccessMessage';
+import BeanItemImage from '../browse/BeanItemImage';
+
+class OrderEdit extends Component {
```

Comment:
I don't think we need this component at all. We should have a 'PackagingInfoContainer' and 'PackagingInfo' component instead.